tanjun/traits.py

```diff
@@ -374,6 +406,16 @@ def names(self) -> typing.Sequence[str]:
     def names(self, names: typing.Sequence[str], /) -> None:
         raise NotImplementedError
 
+    def add_converter(
+        self, converter: typing.Union[typing.Callable[[str], typing.Any], Converter[typing.Any]], /
+    ) -> None:
+        raise NotImplementedError
+
+    def remove_converter(
+        self, converter: typing.Union[typing.Callable[[str], typing.Any], Converter[typing.Any]], /
+    ) -> None:
+        raise NotImplementedError
+
     def bind_component(self, component: Component, /) -> None:
```

Comment:
Is binding component or client for parsing logic necessary or should they be completely separate